pages/_document.js

```diff
@@ -1,11 +1,15 @@
+/*
+ * HTML document
```

Comment:
again, please let's not do comments like this

Reply:
Removed